docs/turn-howto.md

```diff
@@ -11,6 +11,12 @@ TURN server.
 
 The following sections describe how to install [coturn](<https://github.com/coturn/coturn>) (which implements the TURN REST API) and integrate it with synapse.
 
+## requirements
+
+For TURN relaying with `coturn` to work, it must be hosted on a server/endpoint with a public IP.
+
+Hosting TURN behind a NAT (even with appropriate pirt forwarding) is known to cause issues and often not work.
```

Comment:
```suggestion
Hosting TURN behind a NAT (even with appropriate port forwarding) is known to cause issues
and to often not work.
```

Do you have a source for this?

Reply:
I'd rather think that the whole point of a TURN server is that it has a public IP address. The idea is that it provides a public relay that two clients, both behind NAT, can punch out towards. Obviously if the TURN server itself is behind NAT, it's not adding much to the situation.

icbw though, it's a while since I've looked at this stuff...